src/components/UserProfile.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div class="w-full bg-amber-400 p-8">
+    <div class="bg-white rounded p-16 flex justify-between gap-44" v-show="!showQR">
```

Comment:
Try putting v-show/ v-ifs etc as the first attributes

Reply:
https://vuejs.org/guide/essentials/conditional.html

1. Try using v-if/v-else constructs
2. Using positive conditionals (this may be out of context, but still helpful)

https://dev.to/somedood/please-dont-write-confusing-conditionals-2n32